### PR TITLE
Back to attributes link on attributes edit page

### PR DESCRIPTION
**Pull request by AnwerAR**
see #15059 The issue was about editing product attributes. Added "Back to attributes" link.

### CODE REVIEW

File: includes/admin/class-wc-admin-attributes.php [hunk 1]
```diff
@@ -202,7 +202,7 @@ private static function process_edit_attribute() {
 			);
 		}
 
-		echo '<div class="updated"><p>' . __( 'Attribute updated successfully', 'woocommerce' ) . '</p></div>';
+		echo '<div class="updated"><p>' . __( 'Attribute updated successfully', 'woocommerce' ) . '</p><p><a href="'. admin_url( 'edit.php?post_type=product&amp;page=product_attributes' ).'">&larr; ' . __( 'Back to Attributes', 'woocommerce' ) . '</a></p></div>';
```
Comment (mikejolley) [inline includes/admin/class-wc-admin-attributes.php]: Wrap URL in `esc_url`

Comment (mikejolley) [inline includes/admin/class-wc-admin-attributes.php]: Also needs space here `).'"`

**Review by mikejolley — CHANGES_REQUESTED** [review]

File: includes/admin/class-wc-admin-attributes.php (hunk 1 above)
Comment (AnwerAR) [inline includes/admin/class-wc-admin-attributes.php]: Should i replace `admin_url` with `esc_url` or can wrap `admin_url` into `esc_url`?

**Comment by mikejolley** [comment]
Not sure what you're doing here... ecs_url is not valid, and I mean't WRAP the admin_url with an escaping function.

**Comment by mikejolley** [comment]
Same typo. `ecs_url`?

**Comment by claudiosanches** [comment]
We need to avoid using arrows `&larr; `, this is bad for RTL languages, even worse that is not able not even to translate it different.

**Comment by AnwerAR** [comment]
@mikejolley OMG, sorry for typo. it's correct on my localhost. but i do not push changes from localhost. wanna understand `Grunt` tasks first. 
@claudiulodro shall i completely remove `&larr;` or need to be wrapped in `gettext` function along with message? what about `is_rtl() ? '&larr;' : '&rarr;';`?

**Pull request merged**